apps/src/netsim/netsim.js

```diff
@@ -304,7 +322,9 @@ NetSim.prototype.changeRemoteDnsMode = function (newDnsMode) {
  * @param {boolean} isDnsNode
  */
 NetSim.prototype.setIsDnsNode = function (isDnsNode) {
-  this.tabs_.setIsDnsNode(isDnsNode);
```

Comment:
seeing setIsDnsNode here calling another setIsDnsNode in another class makes me wonder if there's an alternate way to do this, but it's just a feeling, nothing concrete.  (can tabs be listening for a similar state change?)